.github/workflows/cf-deploy.yml

```
@@ -28,6 +28,9 @@ jobs:
           elif [ "${{ github.ref }}" = "refs/heads/master" ]; then
             echo "::set-output name=env_name::staging"
             echo "::set-output name=build_script::npm run build-staging"
+          elif [ "${{ github.ref }}" = "refs/heads/DOI-2408-gp4btc-staging" ]; then
```

@Timtech4u Would it be possible to remove handling deployments from repository? I would like to just deploy docker images and that's all. Everything else (where it will be deployed) will be handled by you.

```suggestion
          elif [ "${{ github.ref }}" = "refs/heads/gp4btc-staging" ]; then
```

---

@dawidgil  are you referring to the outdated commit or generally? More context would be helpful

---

No, I wrote nothing about commit. I'm talking about deployment switchboard.
What would happen if we will have 4 more instances of switchboard in different places? You will add more branches to handle that and they should be deployed under different circumstances?
for example: develop, master, staging, staging-2, staging-3, ..., staging-x, release ?
I would like to have develop, master and release. Nothing more. 
To make that working i could just send somewhere docker images and you automatically via k8s deploy to proper instances.

---

@dawidgil  I think it's fine to refactor or close this PR based on the approach discussed from Slack. cc: @jrhender